README.md

```diff
@@ -70,6 +72,4 @@ Give a ⭐️ if you like this project!
 
```

Comment:
- [ ] Great job so far, one of the requirement is that it contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio), kindly add that.